subsys/bluetooth/mesh/cfg_cli.c

```diff
@@ -70,17 +70,13 @@ static int comp_data_status(struct bt_mesh_model *model,
 
 static int state_status_u8(struct bt_mesh_model *model,
 			   struct bt_mesh_msg_ctx *ctx,
-			   struct net_buf_simple *buf,
+			   uint8_t status,
 			   uint32_t expect_status)
 {
 	uint8_t *param;
-	uint8_t status;
-
-	BT_DBG("net_idx 0x%04x app_idx 0x%04x src 0x%04x len %u: %s",
-	       ctx->net_idx, ctx->app_idx, ctx->addr, buf->len,
-	       bt_hex(buf->data, buf->len));
 
-	status = net_buf_simple_pull_u8(buf);
+	BT_DBG("net_idx 0x%04x app_idx 0x%04x src 0x%04x status 0x%04x",
```

Comment:
This function always returns `0`. So I guess it could still pull status from the message, but return it instead of just `0`.

Reply:
yep. it was my idea too.